filer/management/commands/import_files.py

```diff
@@ -108,27 +107,30 @@ def walker(self, path=None, base_folder=None):
             print(('folder_created #%s / file_created #%s / ' + 'image_created #%s') % (self.folder_created, self.file_created, self.image_created))
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
     """
     Import directory structure into the filer ::
 
         manage.py --path=/tmp/assets/images
         manage.py --path=/tmp/assets/news --folder=images
     """
-
-    option_list = BaseCommand.option_list + (
-        make_option('--path',
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--path',
             action='store',
             dest='path',
             default=False,
```

Comment:
Can I ask you to mark path as required? This https://github.com/divio/django-filer/pull/1058/files#diff-361fe6a032f5ff57a38ea06ab3c367a0R83 fixes an issue if path is empty, but it opens the door to an undesired behavior (importing the whole django project root)

Reply:
Should we drop the default then, if its required no need to have a default value?

Reply:
Yes, we can remove it, as it's redundant